### PR TITLE
Yandex testing

### PR DESCRIPTION
**Pull request by iuriestanila**

### CODE REVIEW

File: src/test/java/com/coherent/training/selenium/stanila/Yandex.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    public static final String USERNAME = "moraru.andy";
+    public static final String PASSWORD = "RootRoot5?";
+
+    @BeforeMethod
+    void setUp(){
+        driver =  WebDriverManager.chromedriver().create();
+        softAssert = new SoftAssert();
+    }
+
+    @Test
```
Comment (shukal94) [inline src/test/java/com/coherent/training/selenium/stanila/Yandex.java:25]: if you want to call this function you should not call Test annotation here
you can make method which will be called in login test

**Review by shukal94 — CHANGES_REQUESTED** [review]
make all the methods in test class public
test names should be named corrspondingly what do the validate, e.g. testLoginSuccessfull
make Locator constants `final` and the names for constants should be in uppercase

File: src/test/java/com/coherent/training/selenium/stanila/Yandex.java (hunk 1 above)
Comment (iuriestanila) [reply src/test/java/com/coherent/training/selenium/stanila/Yandex.java:25]: I have modified.

File: src/test/java/com/coherent/training/selenium/stanila/Yandex.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    public static final String PASSWORD = "RootRoot5?";
+
+    @BeforeMethod
+    public void setUp(){
+        driver =  WebDriverManager.chromedriver().create();
+        softAssert = new SoftAssert();
+    }
+
+    @SneakyThrows
+    public void login(){
```
Comment (shukal94) [inline src/test/java/com/coherent/training/selenium/stanila/Yandex.java]: move this method to the bottom of the class for readability

**Review by shukal94 — APPROVED** [review]

**Pull request merged**